core/config/project_settings.h

```diff
@@ -152,6 +152,11 @@ class ProjectSettings : public Object {
 protected:
 	static void _bind_methods();
 
+#ifndef DISABLE_DEPRECATED
+	bool _load_resource_pack_76161(const String &p_pack, bool p_replace_files, int p_offset);
```

Comment:
```suggestion
	bool _load_resource_pack_bind_compat_76161(const String &p_pack, bool p_replace_files, int p_offset);
```
Just minor nit for the established format, optional